package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "data-portal-ui",
-  "version": "1.1.0a1",
+  "version": "1.1.0a2",
```

Comment:
You can keep it at 1.1.0a1 as long as we do not create a release.

Reply:
b91e7e7d8966a24becdbb6326fd7643308d69151